domain/modelconfig/validators/validators_test.go

```diff
@@ -72,7 +72,71 @@ func (_ *validatorsSuite) TestCharmhubURLNoChange(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
-func (_ *validatorsSuite) TestSpaceCheckerFound(c *gc.C) {
+func (*validatorsSuite) TestAgentVersionChanged(c *gc.C) {
```

Comment:
We seem to be missing the authorised keys validator test

Reply:
Done & Done